MiniApp/Classes/core/Downloader/MiniAppClient.swift

```diff
@@ -37,7 +37,7 @@ internal class MiniAppClient: NSObject, URLSessionDownloadDelegate {
         self.environment.customProjectId = config?.rasProjectId
         self.environment.customSubscriptionKey = config?.subscriptionKey
         self.environment.customAppVersion = config?.hostAppVersion
-        self.environment.customIsPreviewMode = config?.isPreviewMode ?? true
```

Comment:
Removing the boolean value here instead of changing it to false. So that the order of priority will be like below,
* RMAIsPreviewMode
* isPreviewMode (from config is provided)
* Default fallback (i.e Publish mode)